docs/install/index.md

```diff
@@ -703,7 +703,7 @@ brew install openblas@0.3.1
 Add a soft link to the OpenBLAS installation. This example links the 0.3.1 version:
 
 ```bash
-ln -sf /usr/local/opt/openblas/lib/libopenblasp-r0.3.* /usr/local/opt/openblas/lib/libopenblasp-r0.3.1.dylib
+ln -sf /usr/local/opt/openblas/lib/libopenblasp-r0.3.*.dylib /usr/local/opt/openblas/lib/libopenblasp-r0.3.1.dylib
```

Comment:
@ankkhedia @anirudhacharya why openblas@0.3.1? there are bug fixes/performance improvements post this version https://github.com/xianyi/OpenBLAS/releases

Usually you create symlink without the version to a specific version something like this. if there are multiple 0.3* version, the command fails
```
ln -sf /usr/local/opt/openblas/lib/libopenblasp-r0.3.1.dylib /usr/local/opt/openblas/lib/libopenblasp.dylib
``

Reply:
it picks up the BLAS version from the machine where the R-package was built. We explored not hard-linking to a particular BLAS version, but that is not trivial. Will revisit it again. Either ways, this PR need not be blocked because of that right?